expenses-manager-backend/src/main/java/com/endava/expensesmanager/service/impl/UsersServiceImpl.java

```diff
@@ -64,6 +65,9 @@ public void deleteUser(int id) {
 
     @Override
     public UserDto loginUser(String name) {
+        if (!validateName(name)) {
```

Comment:
de ce ai facut o functie pentru a valida numele in conditiile in care avem o singura validare?